docs/development/releasing.rst

```diff
@@ -236,6 +233,16 @@ packages that use the full bugfix/maintenance branch approach.)
       $ git reset --hard v<version>
       $ git push upstream stable --force
 
+#. Build and test the Astropy wheels.  See the `wheel builder README
```

Comment:
nope, wheel building is before pushing to pypi. there has been sometimes issues uncovered by it, so I'm uncomfortable doing anything like this after pypi release.

Reply:
I moved it here because of the comment of "moving the wheel stuff after the tagging". Did I move it too far down? The numbering doesn't match anymore between the original issue and the doc, so I am just guessing here.

Reply:
yes, tagging, wheels and the very last step is pypi upload (and then conda-forge).

Reply:
OK moving it back